### PR TITLE
Enable lint and test workflows

### LINKED ISSUE #2: Install Github Actions `test` workflows

### LINKED ISSUE #3: Install Github Actions `lint` workflows

### PR DESCRIPTION
**Pull request by mbestavros**
Closes #2 and closes #3.

Signed-off-by: Mark Bestavros <mbestavr@redhat.com>

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+name: lint
+jobs:
+  fmt:
+    name: cargo fmt
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          target: x86_64-unknown-linux-musl
```
Comment (haraldh) [inline .github/workflows/lint.yml]: I think musl is not required for rustfmt

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  clippy:
+    name: cargo clippy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: sudo apt update
+      - run: sudo apt install -y musl-tools
+      - uses: actions-rs/toolchain@v1
+        with:
+          target: x86_64-unknown-linux-musl
```
Comment (haraldh) [inline .github/workflows/lint.yml]: also not for clippy

File: .github/workflows/lint.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+          args: -- -D warnings
+
+  readme:
+    name: cargo readme
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          target: x86_64-unknown-linux-musl
```
Comment (haraldh) [inline .github/workflows/lint.yml]: also not for readme

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    name: ${{ matrix.toolchain }} (${{ matrix.profile.name }})
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: sudo apt update
+      - run: sudo apt install -y musl-tools
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.toolchain }}
+          target: x86_64-unknown-linux-musl
```
Comment (haraldh) [inline .github/workflows/test.yml]: and I think also not for compiling

**Review by haraldh — DISMISSED** [review]

**Comment by mbestavros** [comment]
Removed the `musl` targets, per @haraldh.

I've also added `deny.toml` in order for `cargo deny` to work.

**Review by haraldh — APPROVED** [review]

**Review by lkatalin — APPROVED** [review]

**Pull request merged**